src/SourceBuild/tarball/content/eng/Versions.props

```diff
@@ -21,8 +21,8 @@
   </PropertyGroup>
   <!-- Production Dependencies -->
   <PropertyGroup>
-    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-6.0.100-bootstrap.19</PrivateSourceBuiltArtifactsPackageVersion>
+    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-6.0.100-bootstrap.21</PrivateSourceBuiltArtifactsPackageVersion>
```

Comment:
For my knowledge - why is this duplicated with `eng/Versions.props`?

Reply:
Interesting that you ask, I am actively working on another issue to reduce the source built artifacts size and I don't think this is referenced.  I will clean this up in that PR.